QEfficient/transformers/modeling_utils.py

```diff
@@ -115,6 +138,15 @@
     MistralForCausalLM: QEffMistralForCausalLM,
     MistralRotaryEmbedding: QEffMistralRotaryEmbedding,
     MistralRMSNorm: CustomRMSNormAIC,
+    # Mixtral model layers
+    MixtralAttention: QEffMixtralAttention,
+    MixtralModel: QEffMixtralModel,
+    MixtralDecoderLayer: QEffMixtralDecoderLayer,
+    MixtralForCausalLM: QEffMixtralForCausalLM,
+    MixtralRotaryEmbedding: QEffMixtralRotaryEmbedding,
+    MixtralRMSNorm: QEffMixtralRMSNorm,
```

Comment:
This should be mapped to same `CustomRMSNormAIC `right? is there any difference in Mixtral `RMSNorm`?